lexer/impl/lexer.cpp

```diff
@@ -1,6 +1,97 @@
 #include "lexer.hpp"
+#include "token.hpp"
+#include <vector>
+#include <string>
+#include "trie.hpp"
+#include <iostream>
+#include <fstream>
+using namespace std;
 
-int sum(int a, int b) 
+common::Trie<int> initTrie()
 {
```

Comment:
Our informal style-guide asks you to move this brace to the line with the function definition.